mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -885,7 +892,109 @@ static int emsmdbp_copy_properties(struct emsmdbp_context *emsmdbp_ctx, struct e
 				SRow_addprop(aRow, newValue);
 			}
 		}
-		if (emsmdbp_object_set_properties(emsmdbp_ctx, dest_object, aRow) != MAPISTORE_SUCCESS) {
+
+		ret = emsmdbp_object_set_properties(emsmdbp_ctx, dest_object, aRow);
```

Comment:
Shouldn't it be `enum mapistore_error` type? Sorry I don't see that in the previous review.